app/Http/Controllers/Api/ContestsController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Http\Transformers\ContestTransformer;
+
+class ContestsController extends ApiController
+{
+    /**
+     * @var \Gladiator\Http\Transformers\UserTransformer
+     */
+    protected $transformer;
+
+    /**
+     * Create a new ContestsController instance.
+     *
+     * @param \Gladiator\Http\Transformers\ContestTransformer $transformer
+     */
+    public function __construct(ContestTransformer $transformer)
```

Comment:
Yeaaaa, I kinda prefer that. I just checked in Northstar and we do it both ways though. :joy: